File: PHP/Tarjeta.php
<?php
class Tarjeta extends Payment{
    public $tipoTarjeta;
    public $banco;

    public function __construct($id, $tipoTarjeta, $banco){
        parent ::__construct ($id);
        $this ->$tipoTarjeta = $tipoTarjeta;
        $this ->$banco = $banco;

    }
}
?>

File: Python/tarjeta.py
from payment import Payment

class Tarjeta(Payment):
    tipoTarjeta = str
    banco = str

    def __init__(self, id, tipoTarjeta, banco):
        super.__init__(id)
        self.tipoTarjeta = tipoTarjeta
        self.banco = banco